src/systems/systems/src/hydraulic/mod.rs

```diff
@@ -1 +1,1720 @@
+            * (indicated_speed.get::<knot>() * indicated_speed.get::<knot>() / 100.)
+            * 0.5
+            * (std::f64::consts::PI / 2. * stow_pos).sin();
+        self.torque_sum += air_speed_torque;
+    }
+
+    fn update_friction_torque(&mut self, displacement_ratio: f64) {
+        let mut pump_torque = 0.;
+        if self.rpm < Self::LOW_SPEED_PHYSICS_ACTIVATION {
+            pump_torque += (self.position * 4.).cos() * displacement_ratio.max(0.35) * 35.;
+            pump_torque += -self.speed * 15.;
+        } else {
+            pump_torque += displacement_ratio.max(0.35) * 1. * -self.speed;
+        }
+        pump_torque -= self.speed * 0.05;
+        // Static air drag of the propeller
+        self.torque_sum += pump_torque;
+    }
+
+    fn update_physics(&mut self, delta_time: &Duration) {
+        self.acceleration = self.torque_sum / Self::PROPELLER_INERTIA;
+        self.speed += self.acceleration * delta_time.as_secs_f64();
+        self.position += self.speed * delta_time.as_secs_f64();
+
+        // rad/s to RPM
+        self.rpm = self.speed * 30. / std::f64::consts::PI;
+
+        // Reset torque accumulator at end of update
+        self.torque_sum = 0.;
+    }
+
+    fn update(
+        &mut self,
+        delta_time: &Duration,
+        indicated_speed: &Velocity,
+        stow_pos: f64,
+        displacement_ratio: f64,
+    ) {
+        if stow_pos > 0.1 {
+            // Do not update anything on the propeller if still stowed
+            self.update_generated_torque(indicated_speed, stow_pos);
+            self.update_friction_torque(displacement_ratio);
+            self.update_physics(delta_time);
+        }
+    }
+}
+impl SimulationElement for WindTurbine {
+    fn write(&self, writer: &mut SimulatorWriter) {
+        writer.write_f64("HYD_RAT_RPM", self.rpm());
+    }
+}
+impl Default for WindTurbine {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+struct AlwaysPressurisePumpController {}
+impl AlwaysPressurisePumpController {
+    fn new() -> Self {
+        Self {}
+    }
+}
+impl PumpController for AlwaysPressurisePumpController {
+    fn should_pressurise(&self) -> bool {
+        true
+    }
+}
+impl Default for AlwaysPressurisePumpController {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+pub trait RamAirTurbineController {
+    fn should_deploy(&self) -> bool;
+}
+
+pub struct RamAirTurbine {
+    deployment_commanded: bool,
+    pump: Pump,
+    pump_controller: AlwaysPressurisePumpController,
+    wind_turbine: WindTurbine,
+    position: f64,
+    max_displacement: f64,
+}
+impl RamAirTurbine {
+    const DISPLACEMENT_BREAKPTS: [f64; 9] = [
+        0.0, 500.0, 1000.0, 1500.0, 2100.0, 2300.0, 2600.0, 3050.0, 3500.0,
+    ];
+    const DISPLACEMENT_MAP: [f64; 9] = [1.15, 1.15, 1.15, 1.15, 1.15, 0.5, 0.0, 0.0, 0.0];
+
+    // 1 == no filtering. !!Warning, this will be affected by a different delta time
+    const DISPLACEMENT_DYNAMICS: f64 = 0.2;
```

Comment:
This is a low pass filter, that when used in a fixed step context is as simple to implement as output=0.1*new_value+0.9*old_value.
Different fixed delta would change filter stop frequency. Variable delta would spice things up :D. Hence the comment.
Fixed frequency time dependent formula is less straight forward so I only used it in brakes where it might end up being refreshed at sim rate.
I'll update comment anyway to make clear it's an example, not an old comment left with wrong value.